spec.md

```diff
@@ -423,13 +423,17 @@ Manifest byte stream:
 
 The uploaded manifest MUST reference any blobs that make up the artifact.
 However, the list of blobs MAY be empty.
+
+The registry MUST store the manifest in the exact byte representation provided by the client.
 Upon a successful upload, the registry MUST return response code `201 Created`, and MUST have the following header:
 
 ```
 Location: <location>
 ```
 
 The `<location>` is a pullable manifest URL.
+The Docker-Content-Digest header returns the canonical digest of the uploaded blob, and MUST be equal to the client provided digest.
```

Comment:
suggest rewording this to fit the paragraph..

what if the digest differ? What if the client did not provide one..